### PR TITLE
Adds performance suite

### PR DESCRIPTION
**Pull request by jruere**
Very basic operations are timed and compared among several
implementations.

PS: your library is awesome!

### CODE REVIEW

File: performance_suites/benchmark.py [hunk 1]
```diff
@@ -0,0 +1,137 @@
+import timeit
```
Comment (1st1) [inline performance_suites/benchmark.py:1]: We should use https://pyperf.readthedocs.io/ that produces way more stable and accurate results.

**Comment by 1st1** [comment]
> PS: your library is awesome!

Thanks! I suggest to rework the PR to use pyperf.

**Comment by jruere** [comment]
pyperf is nice but hard to use programmatically. I'll need some time.

**Comment by jruere** [comment]
I had completely forgotten about this. Sorry, I don't want to improve it.

**Pull request closed without merging**